app/views/propertyrepresentation/revokeAgentProperties.scala.html

```diff
@@ -129,165 +128,151 @@
 
 @mainLayout(pageTitle = title, backUri = Some(backLink), mainClass = Some("govuk-grid-column-full"), hasErrors = errorLinks.nonEmpty) {
 
-    <div class="govuk-grid-row">
-
-        @if(f.isDefined){
-            @govukErrorSummary(
-                ErrorSummary(
-                    errorList = errorLinks,
-                    title = Text(messages("error.title"))
-                )
+    @if(f.isDefined){
+        @govukErrorSummary(
+            ErrorSummary(
+                errorList = errorLinks,
+                title = Text(messages("error.title"))
             )
-        }
-
-        <h1 class="govuk-heading-l">@messages("propertyRepresentation.removeAgentProperties.heading")</h1>
-
-        <p class="govuk-body">@messages("propertyRepresentation.removeAgentProperties.message.p1", model.agentGroup.companyName, agentCode.toString)</p>
-
-        @formWithCSRF(agentAppointment.routes.AppointAgentController.filterPropertiesForRevoke(pagination, agentCode)) {
-            <input type="hidden" name="pageSize" value="@pagination.pageSize">
+        )
+    }
 
-            <input type="hidden" name="backLink" value="@backLink">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">@title</h1>
+
+            <p class="govuk-body">
+                <span class="govuk-!-font-weight-bold">@model.agentGroup.companyName</span>
+                @messages("propertyRepresentation.removeAgentProperties.message", agentCode.toString)
+            </p>
+            <br/>
+        </div>
+    </div>
 
+    @formWithCSRF(agentAppointment.routes.AppointAgentController.filterPropertiesForRevoke(pagination, agentCode)) {
+        <input type="hidden" name="pageSize" value="@pagination.pageSize">
+        <input type="hidden" name="backLink" value="@backLink">
+
+        <h2 class="govuk-heading-m">@messages("propertyRepresentation.search.filter")</h2>
+        @govukInput(Input(
+            id = "address",
+            name = "address",
+            value = params.address,
+            label = Label(
+                content = Text(messages("propertyRepresentation.th.ADDRESS"))
+            ),
+            classes = "govuk-!-width-one-third"
+        ))
+
+        <div class="govuk-button-group">
+            @govukButton(Button(
+                content = Text(messages("label.search"))
+            ))
+            <a href="@agentAppointment.routes.AppointAgentController.selectAgentPropertiesSearchSort(PaginationParameters(), agentCode).url" class="govuk-body govuk-link">@messages("manageProperties.table.clear.search")</a>
+        </div>
```

Comment:
I think it is better to have the class `govuk-body` on the surrounding div, not on the <a>, if it is needed at all. I'm not sure, but that's would I would have done I think

Reply:
I'll take a look at changing this and the tests, thanks